tools/lib/test_script.py

```diff
@@ -120,6 +122,9 @@ def find_js_test_files(test_dir: str, files: Iterable[str]) -> List[str]:
 
 def prepare_puppeteer_run(is_firefox: bool = False) -> None:
     os.chdir(ZULIP_PATH)
+    download_path = platformdirs.user_cache_dir("zulip/puppeteer-download")
+    os.makedirs(download_path)
+    os.environ["PUPPETEER_DOWNLOAD_PATH"] = download_path
```

Comment:
Does this need to be in `gitignore`, or inside the `var/` subdirectory intended for that sort of thing?

Reply:
This is `~/.cache/zulip/puppeteer-download` (unless you customized `$XDG_CACHE_HOME`; [platformdirs documentation](https://platformdirs.readthedocs.io/en/latest/api.html#platformdirs.unix.Unix.user_cache_dir)).

I didn’t put this in `var/` because I wasn’t excited about discovering funny side effects of running a Chromium installation out of the VirtualBox shared filesystem—it’s probably slow at best.